ChiTech/ChiMesh/MeshContinuum/chi_meshcontinuum_exportvtk.cc

```diff
@@ -72,22 +62,20 @@ void chi_mesh::MeshContinuum::ExportCellsToVTK(const char* baseName)
       std::vector<vtkIdType> cell_info(num_verts);
       for (int v=0; v<num_verts; v++)
       {
-        int vgi = slab_cell->vertex_ids[v];
+        uint64_t vgi = slab_cell->vertex_ids[v];
```

Comment:
vgi = vertex global ID? If so, can you add a comment where appropriate(or rename, if appropriate)?

Reply:
Comment added at line.